deployer/templates/es.yaml

```diff
@@ -41,6 +41,7 @@ objects:
             component: ${ES_CLUSTER_NAME}
             deployment: ${ES_DEPLOYMENT_NAME}
         spec:
+          terminationGracePeriodSeconds: 1800
```

Comment:
oc delete pod goes through the same termination grace period, there's no
"force-delete" other than kill -9 on the PID. 10 is good, LGTM

On Mon, Sep 12, 2016 at 10:24 AM, Eric Wolinetz <notifications@github.com>
wrote:

> In deployer/templates/es.yaml
> <https://github.com/openshift/origin-aggregated-logging/pull/227#discussion_r78382047>
> :
>
> > @@ -41,6 +41,7 @@ objects:
> >              component: ${ES_CLUSTER_NAME}
> >              deployment: ${ES_DEPLOYMENT_NAME}
> >          spec:
> > +          terminationGracePeriodSeconds: 1800
>
> We can drop it down to 10 minutes. I was planning for a worst case where
> they have a ton of indices that the metadata needed to be written out for
> and they could always oc delete the pod if its taking too long. Since we
> don't have actual performance number on this it was kind of a guess
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/227/files/0a5f7c5388b8c07046cadf28009322c7f2dce887#r78382047>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABz-hfniy5Y-Q8XWQZ3iCedCoO6YMwMks5qpWCcgaJpZM4J4doN>
> .
>